build/node-feature-discovery/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM golang:1.13.5-alpine as build
+
+ARG VERSION=v0.6.0
+ARG TARGETPLATFORM
+ENV GO111MODULE=on \
+    CGO_ENABLED=0
+
+RUN apk add make git
+
+WORKDIR /opt
+RUN git clone --depth 1 -b ${VERSION} https://github.com/kubernetes/node-feature-discovery.git . \
+    && export GOOS=$(echo ${TARGETPLATFORM} | cut -d / -f1) \
+    && export GOARCH=$(echo ${TARGETPLATFORM} | cut -d / -f2) \
+    && GOARM=$(echo ${TARGETPLATFORM} | cut -d / -f3); export GOARM=${GOARM:1}
+
+# Get (cache) deps in a separate layer
+COPY go.mod go.sum /go/node-feature-discovery/
+
+WORKDIR /go/node-feature-discovery
```

Comment:
If you move the `WORKDIR` above the `COPY` you can update the COPY to the following:

```
COPY go.mod go.sum .
```

Although I'm assuming you already have them within the image from the git clone and don't need to COPY them into the image.